geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/BoundingBox.scala

```diff
@@ -72,6 +72,14 @@ case class BoundingBox(ll: Point, ur: Point) {
    */
   def contains(otherGeom: Geometry): Boolean = geom.contains(otherGeom)
 
+  /**
+   * This bounding box equals bbox if the lower left and upper right points are the same.
+   * @param bbox
+   * @return
+   */
+  def equals(bbox: BoundingBox): Boolean = (ll.getX == bbox.ll.getX) && (ll.getY == bbox.ll.getY) &&
```

Comment:
I am not sure how to address this yet... The BoundingBox uses two Point objects to denote lower left and upper right coordinates. Those Points in turn store their information in a Coordinate object, which uses doubles. For the purposes of equals as used here, I think that *strictness* is required. Anything else would be an "approximately equals" or an "equals within tolerance". I think that this may justify a new ticket.